build.gradle

```diff
@@ -8,6 +8,10 @@ buildscript
   }
 }
 
+plugins {
+  id 'org.gradle.test-retry' version '1.1.5' apply false
```

Comment:
Out of curiosity: Why is it necessary to declare this in the root build script? Can't you just import the plugin in the stf build script?

Reply:
It is not necessary as long as we use the plugin only in one subproject. (Define the dependency once in the root project and using it by other subprojects).
IIRC it was necessary for a previous gradle version, but not anymore.